File: api/routes/wishlist.route.js
import express from "express";
import {
  addWishlist,
  displayWishlist,
  removeWishlist,
} from "../controllers/wishlist.controller.js";

const router = express.Router();

router.post("/addWishlist", addWishlist);
router.get("/displayWishlist", displayWishlist);
router.delete("/removeWishlist", removeWishlist);

export default router;

File: api/routes/wishlist.route.go
package routes

import (
	"github.com/gin-gonic/gin"
	"api/controllers"
)

func WishlistRoutes(router *gin.Engine) {
	wishlist := router.Group("/api/wishlist")
	{
		wishlist.POST("/addWishlist", controllers.AddWishlist)
		wishlist.GET("/displayWishlist", controllers.DisplayWishlist)
		wishlist.DELETE("/removeWishlist", controllers.RemoveWishlist)
	}
}
